### PR TITLE
Close #352 - Fix old IS_IN_DB syntax

### LINKED ISSUE #352: Correct IS_IN_DB() old syntax in example
Here : http://www.web2py.com/books/default/chapter/29/07/forms-and-validators#Links-to-referencing-records

In example section IS_IN_DB() is defined like this : 
```python
db.dog.owner.requires = IS_IN_DB(db,db.person.id,'%(name)s')
```

But this seems to be a wrong way of doing it as it seems when setting validator that way it prevent the referenced table to be lazy in context of lazy_tables = True

We should correct the book or mention the drawback of such kind of syntax...

Should be:
```python
db.dog.owner.requires = IS_IN_DB(db, 'person.id', '%(name)s')
```
...

­­­

### PR DESCRIPTION
**Pull request by BuhtigithuB**
And improve spacing between attributes here and there...

### CODE REVIEW

File: sources/29-web2py-english/07.markmin [hunk 1]
```diff
@@ -763,7 +763,7 @@ db.define_table('person',
 db.define_table('dog',
     Field('owner', 'reference person'),
     Field('name', requires=IS_NOT_EMPTY()))
-db.dog.owner.requires = IS_IN_DB(db,db.person.id,'%(name)s')
+db.dog.owner.requires = IS_IN_DB(db, 'db.person.id', '%(name)s')
```
Comment (mdipierro) [inline sources/29-web2py-english/07.markmin]: I will take the PR but can you fix this. It should be without quotes or just 'person.id' in quotes.

Comment (BuhtigithuB) [inline sources/29-web2py-english/07.markmin]: Yes my bad...

**Comment by BuhtigithuB** [comment]
I make another PR, I had so much issue with this old branch and unsynched master in my local web2py-book clone... I include more improvement explain in the PR message. Thanks.

**Comment by mdipierro** [comment]
thanks

**Pull request closed without merging**